pgxn/neon/libpagestore.c

```diff
@@ -61,23 +64,63 @@ int			flush_every_n_requests = 8;
 int			n_reconnect_attempts = 0;
 int			max_reconnect_attempts = 60;
 
+#define MAX_PAGESERVER_CONNSTRING_SIZE 256
+
+typedef struct
+{
+    LWLock lock;
```

Comment:
I do not see place this `lock` is initialized.
Please look at file_cache.c for example of `lfc_lock` registration and initialization.

Reply:
Thank you, I wasn't sure how to initialize it. I believe I fixed it